bot/commands/misc/roll.js

```diff
@@ -0,0 +1,25 @@
+// Rolls between 2 numbers provided or a default 0-6
+
+const {Command} = require('yuuko');
+
+function getRandomInt (min, max) {
+	min = Math.ceil(min);
+	max = Math.floor(max);
+	return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
+module.exports = new Command('roll', (msg, args) => {
+	let min = 0;
+	let max = 6;
+	if (args.length !== 0) {
+		min = args[0];
+		max = args[1];
```

Comment:
Use `parseInt(thing)` to explicitly convert strings to numbers, otherwise you'll get unexpected values with bad input. You might also want to check `isNaN(thing)` after converting to number and throw an error in that case. (`NaN` is returned by `parseInt()` if the string doesn't represent a number. For weird javascript reasons, you can't compare against `NaN` using normal equality operators.)

Also, similar to the command above, you handle the no argument case, but if the user passes one argument you'll end up with `max` being `undefined` which might cause unexpected results.